dagger/config_finder/config_processor.py

```diff
@@ -29,6 +33,14 @@ def _load_yaml(yaml_path):
                 exit(1)
         return config
 
+    @staticmethod
+    def overwrite_params(config):
```

Comment:
Can we give a more descriptive name? localising_params maybe or something that expresses we are modifying based on environment